_layouts/default.html

```diff
@@ -13,17 +13,22 @@
 
     <header>
       <div class="container">
-        <a id="a-title" href="{{ '/' | relative_url }}">
-          <h1>{{ site.title | default: site.github.repository_name }}</h1>
+        <h1>{{ site.title | default: site.github.repository_name }}</h1> 
+        <a href="{{ site.github.repository_url }}" class="btn btn-github" style="float:right">
+          <span class="icon"></span>
+          View on GitHub
         </a>
         <h2>{{ site.description | default: site.github.project_tagline }}</h2>
 
-        <section id="downloads">
+        <section id="downloads" float="left">
           {% if site.show_downloads %}
             <a href="{{ site.github.zip_url }}" class="btn">Download as .zip</a>
             <a href="{{ site.github.tar_url }}" class="btn">Download as .tar.gz</a>
           {% endif %}
-          <a href="{{ site.github.repository_url }}" class="btn btn-github"><span class="icon"></span>View on GitHub</a>
+
+          {% unless page.hide_navigation == true %}
+            <a href="{{ site.url }}">Home</a> | <a href="{{site.url}}\another-page\">Another Page</a>
```

Comment:
```suggestion
            <a href="{{ site.url }}">Home</a>
```

Fixes failing tests, & removes unnecessary hint.